src/feature.js

```diff
@@ -354,27 +323,33 @@ class Feature {
   // The refresh count is retained across sessions - ex if they refresh once
   // then close the browser and later refresh on the same page, that page's count
   // will be retained and incremented.
-  possiblyShowNotification() {
-    const reloadCount = this.fastblockRecords[this.payload.etld].count;
+  possiblyShowNotification(tabInfo) {
+    if (tabInfo.surveyShown) {
+      return;
+    }
+
     const num = Math.floor(Math.random() * 10);
-    if (num <= (3 + reloadCount)) {
-      this.fastblockRecords[this.payload.etld].survey_shown = true;
+    if (num <= (3 + tabInfo.reloadCount)) {
+      tabInfo.page_reloaded_survey = SURVEY_SHOWN;
+      tabInfo.surveyShown = true;
```

Comment:
In this case line 333 is actually wrong, it should be `tabInfo.telemetryPayload.page_reloaded`, and since that is reset on any navigation I will have to keep using `surveyShown`.